pkg/sdn/api/types.go

```diff
@@ -44,7 +44,7 @@ type NetNamespace struct {
 	kapi.ObjectMeta
 
 	NetName string
-	NetID   uint
+	NetID   *uint
```

Comment:
okay, I will change v1 and v1beta3 api to *uint and we no longer need any of these UnsetVNID/conversion functions.